governance.md

```diff
@@ -109,7 +118,7 @@ Current SC members:
 * Ruslan Meshenberg (Netflix)
 * Steven Kim (Google)
 
-The steering committee is initially seeded by Netflix and Google as the project matures and its structure stabilizes. We will leave further details such as number of seats, qualifications, tenure, appointing measures, TBD for the time being.
+The SC should remain with an odd number of members so as to avoid deadlock. The SC will nominate a chairperson for a 1 year term. Their duty is to run/call meetings, take meeting notes, and call votes. 
```

Comment:
Any upper or lower limits desired here? I'd say at least 3?

Reply:
Given that the Steering Committee has the ability to rewrite these guidelines, I am how much restrictive prose to write here.  I would suggest replacing the new text with the following:

The Steering Committee will initially have five members.  The Steering Committee may adjust the size of the committee each year as they deems appropriate, with a bias towards an odd number of members to avoid voting deadlocks.  In the event that someone from the Steering Committee leaves during their term, the Steering Committee shall appoint someone to fill the remainder of the term.  Each year, the Steering Committee will nominate a member to serve as the Chairperson for that year.  The Chair's duties will include calling and runnings meetings, calling votes, and ensuring that meeting notes are recorded.

Reply:
I like the way this reads. I also agree that writing up a bunch of additional constraints doesn't seem worthwhile since as you pointed out, the SC can just change the policy anyway.